### PR TITLE
docs(README): refactor for webpack v2

### PR DESCRIPTION
**Pull request by michael-ciniawsky**
[README](https://github.com/michael-ciniawsky/mocha-loader/tree/master)

@webpack/core-team  @webpack/contributor-team

### CODE REVIEW

**Comment by michael-ciniawsky** [comment]
@JanKanty @SpaceK33z @TheLarkInn  README + JSF License :shipit:

**Comment by TheLarkInn** [comment]
Hey this looks great overall. I'd love to see our Sponsor and Backers information on these readmes too if possible. What do you think @spacek33z

**Comment by michael-ciniawsky** [comment]
@TheLarkInn We could add the badges under the Title and append Sponsers/Backers under Maintainers like in webpack/webpack

```
Tiltle
[Sponsors][Backers]

Maintainers
Sponsors 
Backers
```
Will do, let my relicense the missing repos first 😛

**Comment by TheLarkInn** [comment]
That's fine no worries maybe we'll make a separate issue

File: README.md [hunk 1]
```diff
@@ -1,39 +1,94 @@
...
-    entry: './entry-file.js',
-    output: {
-        path: __dirname,
-        filename: 'bundle.js'
-    }
+  entry: './entry.js',
+  output: {
+    path: __dirname,
+    filename: 'bundle.js'
+  }
```
Comment (jhnns) [inline README.md]: Seems like the mocha-loader is missing here?

File: README.md [hunk 2]
```diff
@@ -1,39 +1,94 @@
...
 
-#### entry-file.js
 ```js
-/*additional setup with other loaders (polyfills, ...)*/
-const context = require.context(/*directory*/'mocha-loader!./tests', /*recursive*/true, /*match files*//_test.js$/);
-context.keys().forEach(context);
+import test from './test'
+```
+
+### CLI
```
Comment (jhnns) [inline README.md]: I don't know if you've discussed this already, but personally I wouldn't list all the ways how to add a loader with webpack. Using the `webpack.config.js` is recommended and all the other configuration possibilities should only be used when really necessary.

Comment (michael-ciniawsky) [inline README.md]: Yep I'm in favour of changing aswell, I have planned to revisit the various README's shaping out remaining inconsistencies anyways, I will take care of the removal if all @webpack/core agree

**Pull request merged**